backend/src/main/java/com/greenfoxacademy/backend/dtos/PetDetailsDto.java

```diff
@@ -0,0 +1,28 @@
+package com.greenfoxacademy.backend.dtos;
+
+import jakarta.validation.constraints.FutureOrPresent;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.PastOrPresent;
+import java.util.Date;
+import lombok.Data;
+
+/**
+ * A data transfer object for pet details.
+ *
+ * @author Your Name
+ */
+@Data
+public class PetDetailsDto {
```

Comment:
Didn't we change this last time?

Reply:
We have changed it in the next ticket which is SCRUM-96.